modules/core/04-channel/keeper/upgrade.go

```diff
@@ -731,7 +731,7 @@ func (k Keeper) checkForUpgradeCompatibility(ctx sdk.Context, proposedUpgradeFie
 	if err != nil {
 		// NOTE: this error is expected to be unreachable as the proposed upgrade connectionID should have been
 		// validated in the upgrade INIT and TRY handlers
-		return errorsmod.Wrap(err, "expected proposed connection to be found")
```

Comment:
i kind of like this error message better tbh because we should actually never reach this error case.

Reply:
I changed to make the error more similar to the one that is returned in other places in `upgrade.go`, like [this one](https://github.com/cosmos/ibc-go/blob/f731de5d9c5f6668049ca9293b1fda46915f0a98/modules/core/04-channel/keeper/upgrade.go#L98). But I can change it back, if you want.